Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php

```diff
@@ -91,7 +91,7 @@ private static function encloseTextInComment($header)
         foreach ($lines as $line) {
             $comment .= rtrim(' * '.$line)."\n";
         }
-        $comment .= " */\n";
+        $comment .= ' */';
```

Comment:
No. If rebase would be needed then the merge wouldn't be possible.

This change was indeed done, but on master branch.